CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -148,6 +148,7 @@ extern    "C"
    * @param[in]       buffer_a    pointer to buffer space used for input optimization(partial im2col) and is necessary
```

Comment:
I wonder if it should be "param[out]" here since the buffer is modified ?

Reply:
Good question, a matter of definition I guess